pkg/crypto/aes.go

```diff
@@ -172,28 +158,53 @@ func decodeHex(passphrase string, ciphertext string) ([]byte, []byte, []byte, er
 	return salt, iv, data, nil
 }
 
-// Decrypt ciphertext with a passphrase
-func Decrypt(passphrase string, ciphertext []byte, encoding string) (string, error) {
+// Decrypt ciphertext with a passphrase.
+// Formats:
+// 1) `encoding:text`
+// 2) `text`
+func Decrypt(passphrase string, ciphertext []byte) (string, error) {
 	var salt []byte
 	var iv []byte
 	var data []byte
-	if encoding == "base64" {
+
+	cParts := strings.SplitN(string(ciphertext), ":", 3)
```

Comment:
~nit: Should this be 2?~

Update: I see why this is 3 now, even though 2 were only required in the first commit 👍